downloader/src/main/kotlin/WorkingTree.kt

```diff
@@ -89,12 +89,14 @@ abstract class WorkingTree(val workingDir: File, val vcsType: VcsType) {
     fun guessRevisionName(project: String, version: String): String {
         if (version.isBlank()) throw IOException("Cannot guess a revision name from a blank version.")
 
-        val versionNames = filterVersionNames(version, listRemoteTags(), project)
+        val remoteTags = listRemoteTags()
+        val versionNames = filterVersionNames(version, remoteTags, project)
+
         return when {
             versionNames.isEmpty() ->
                 throw IOException(
-                    "No matching tag found for version '$version'. Please create a tag whose name "
-                            + "contains the version."
+                    "No matching tag found for version '$version' among tags ${remoteTags.joinToString()}. " +
```

Comment:
Why is it needed to have the tags in the exception?

Reply:
It's not *needed*, but useful to have to debug possible issues in the ORT logic that tries to match version strings to tag names.

Reply:
From my experience with debugging in this area, I think that debugging effort cannot be significantly improved by adding the tags. E.g. the effort for debugging is huge compared to the effort for listing the remote tags manually. This is why I believe it's not needed. I'll leave that up to you however.

Reply:
Please note that when I say "debug" I don't necessarily mean interactive debugging in the IDE, but the whole process of hunting down a bug, which includes reading DEBUG level logs, stack traces etc.

Reply:
Yeah, I used the same for "debug" in this context.

Reply:
So, still good for approval / dismissing your review?

Reply:
yes